op-e2e/interop/supersystem.go

```diff
@@ -409,6 +409,9 @@ func (s *interopE2ESystem) newBatcherForL2(
 		MaxBlocksPerSpanBatch: 10,
 		DataAvailabilityType:  batcherFlags.CalldataType,
 		CompressionAlgo:       derive.Brotli,
+		ThrottleThreshold:     0, // no DA throttling
+		ThrottleInterval:      2 * time.Second,
```

Comment:
Why don't we set a much higher value if we're not throttling? Or we could also use `0` to encode not starting the throttling loop at all, which is fine in tests.

Reply:
I figured if someone forgot to specify the flag, it would default to 0, and it's better to never throttle than always throttle?

Reply:
Changed to make a ThrottleInterval of 0 to mean do-not-start-throttling-loop, and defaulted the threshold flag to a higher non-zero value.